ci/run-migrate

```diff
@@ -83,6 +83,27 @@ check
 ./manage.py shell -c 'from weblate.memory.models import Memory; from weblate.lang.models import Language; Memory.objects.create(source_language=Language.objects.get(code="en"), target_language=Language.objects.get(code="cs"), source="source"*1000, target="target"*1000, origin="origin"*1000)'
 check
 
+# Add a pending unit
+semver_compare "$TAG" "<=" "5.12"
```

Comment:
The TAG is something like `weblate-5.12', so this gets never triggered. Better to use $1 directly. Also, we want to test migrations from version older than 5.12:

```suggestion
semver_compare "$1" "<" "5.12"
```